docs/search/copilot.md

```diff
@@ -0,0 +1,164 @@
+1. Refine query by applying the suggestion `Count logs by eventname, access key and sourceIp`.
+1. Filter for Create and Authorize events, as attackers often create new resources during a breach. We find an anomaly with `AuthoritySecurityGroupIngress` events.
+1. Provide feedback to enhance the AI's accuracy stating that the queries have been accurate so far.  
+1. Add the security `groupid` to the prompt to tabulate events containing that data.
+1. Focus on a specific access key, `ABCDEFGOYCM3PIKNOVRA`, noting `PutRolePolicy` events indicating permission elevation. Multiple AWS accounts are impacted.
+1. We conclude that the access key was used to modify permissions, providing the attacker with a potential network entry point. Remediation would involve disabling the key, blocking the source IP, and further log analysis.
+</details>
+
+## Step 1: Open Copilot
+
+To start using Copilot, navigate to the **Copilot** tab on the Sumo Logic home page.
+
+<img src={useBaseUrl('img/search/copilot/copilot-tab.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="250" />
+
+## Step 2: Select a source category
+
+Click **Select Source Category** and select the data source of the log messages you want to investigate. This field is also called the source expression box.
+
+<img src={useBaseUrl('img/search/copilot/source-category.png')} alt="Copilot source category" style={{border: '1px solid gray'}} width="600" />
+
+## Step 3: Execute a prompt
+
+### Suggestions (recommended)
+
+Under **Suggestions** > **Explore**, click on any of the suggested prompts to refine your investigation. For example:<br/><img src={useBaseUrl('img/search/copilot/suggestions-explore.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
+
+### Manual entry
+
+:::tip
+Because manually typing an AI prompt requires careful precision for optimal performance, we recommend clicking the prebuilt [Suggestions](#suggestions-recommended) prompts, which have been proven effective through extensive testing.
+:::
+
+In the **Ask Something...** field, enter a natural language query prompt similar to the ones under **Suggestions** > **Explore**.
+
+You'll need to be very specific. Broad questions do not return good results. When your question is framed as a query about a small, well-defined problem, Copilot answers more accurately.
+
+:::note
+If the statement in the **Ask Something** field can't be translated into a query, this field will say "Failed translation".
+:::
+
+## Step 4: Refine your investigation
+
+After executing a prompt, you'll see your current investigation summarized in plain text in the **Ask Something...** field. You can use these natural language query prompt ideas to launch and/or refine investigations.
+
+Optionally, follow any of the below steps to refine your search.
+
+### Show/Hide Log Query
+
+Click **Show Log Query** or **Hide Log Query** to show the current investigation as a log query, or hide the query.
```

Comment:
IMO: Show/Hide Log Query is not beneficial until we modify the query to refine our results. Should we merge this section with `Edit query code` ? WDYT ?

Reply:
Done! I agree.